leshan-core/src/main/java/org/eclipse/leshan/core/node/LwM2mSingleResource.java

```diff
@@ -181,6 +181,10 @@ public static LwM2mSingleResource newUnsignedIntegerResource(int id, Long value)
             throw new LwM2mNodeException("Invalid value : positive value expected for UNSIGNED_INTEGER");
     }
 
+    public static LwM2mSingleResource newULongArray(int id, ULong[] value) {
+        return new LwM2mSingleResource(id, value, Type.OPAQUE);
+    }
+
```

Comment:
This sounds no more used, should be deleted.

Do not hesitate to review your own pull request, to find this kind of issue by yourself :wink: